doc/releases/changelog-dev.md

```diff
@@ -2,6 +2,89 @@
 
 <h3>New features since last release</h3>
 
+* The Unfied Compiler, which enables users to develop their own compilation passes using xDSL, has been
+  migrated to Catalyst. It is available under the `catalyst.python_interface` submodule.
+  [(#2199)](https://github.com/PennyLaneAI/catalyst/pull/2199)
+
+  * Add an experimental `outline_state_evolution_pass` xDSL pass to `catalyst.python_interface.transforms`,
```

Comment:
```suggestion
  This functionality was originally developed as part of the PennyLane package, and has been migrated here.
  For earlier development notes to the feature, please refer to the
  [PennyLane release notes](https://docs.pennylane.ai/en/stable/development/release_notes.html#release-0-43-0).
  In addition, the following changes have been made since the last release:

  * Add an experimental `outline_state_evolution_pass` xDSL pass to `catalyst.python_interface.transforms`,
```